tap/parser.py

```diff
@@ -2,6 +2,7 @@
 
 import re
 import sys
+from io import StringIO
```

Comment:
The convention for this project is to alphabetize imports, even when using the `from` form (e.g., `io` above `re`).